bin/normalize-ways

```diff
@@ -78,13 +78,19 @@ waysStream.on('data', function (line) {
       way.properties.id = way.properties.id + ';' + quadkey;
       if (openLR) {
         way.properties.openLR = geojsonLR.toOpenLRString(way);
+        if (way.properties.oneway === 0) {
```

Comment:
would it be better if we make `+way.properties.oneway === 0` or `String(way.properties.oneway) === "0"` ?